config.lua

```diff
@@ -0,0 +1,63 @@
+-- ===================================== General Section ============================================
+lvim.log.level = "warn"
```

Comment:
remove all the extra settings here that don't pertain to the language

Reply:
@ChristianChiarulli Thanks, I have removed it